Modules/StudyProgramme/classes/model/class.ilStudyProgrammeType.php

```diff
@@ -143,6 +143,8 @@ public function __construct($primary_key = 0) {
 		$ilUser = $DIC['ilUser'];
 		$ilPluginAdmin = $DIC['ilPluginAdmin'];
 		$lng = $DIC['lng'];
+		$this->g_webdir = $DIC->filesystem()->web();
+		$this->g_tmpdir = $DIC->filesystem()->temp();
```

Comment:
Please remove the g_-Prefix for the variables, that's internal convention and seems to irritate others.